tox.ini

```diff
@@ -44,56 +46,79 @@ setenv =
     # make sure that directory is accessible by docker
     hive: UPLOAD_FOLDER = /tmp/.superset/app/static/uploads/
 usedevelop = true
-whitelist_externals =
+allowlist_externals =
     npm
+    pkill
 
-[testenv:cypress-dashboard]
+[testenv:cypress]
+setenv =
+    PYTHONPATH = {toxinidir}
+    SUPERSET_TESTENV = true
+    SUPERSET_CONFIG = tests.superset_test_config
+    SUPERSET_HOME = {envtmpdir}
+    ENABLE_REACT_CRUD_VIEWS = true
 commands =
     npm install -g npm@'>=6.5.0'
     pip install -e {toxinidir}/
-    {toxinidir}/superset-frontend/cypress_build.sh dashboard
-deps =
-    -rrequirements/testing.txt
+    {toxinidir}/superset-frontend/cypress_build.sh
+commands_post =
+    pkill -if "python {envbindir}/flask"
```

Comment:
Guessing you've witnessed zombie process issues? Or is it a cypress/tox best practice?

Reply:
Yes, frequent zombie Flask processes when running locally via tox.